core/shared/src/main/scala/zio/ZIO.scala

```diff
@@ -4070,9 +4070,9 @@ object ZIO extends ZIOCompanionPlatformSpecific {
      * is interruptible, then the effect can be interrupted normally, in the
      * foreground.
      */
-    def force[R, E, A](zio: ZIO[R, E, A]): ZIO[R, E, A] =
-      if (flag == InterruptStatus.Uninterruptible) zio.uninterruptible.disconnect.interruptible
-      else zio.interruptStatus(flag)
+    def force[R, E, A](zioF: ZIO[R, E, A]): ZIO[R, E, A] =
```

Comment:
The dotty compiler seemed not able to differentiate between the package  `zio` and the argument name `zio` here. Maybe we should call the `zioF` something else, or is it ok?

Reply:
We could rename it to `effect` to resolve the issue for now but that seems like a pretty serious problem so we should probably open an issue for it with the Dotty team.